qrenderdoc/Widgets/AboutDialog.cpp

```diff
@@ -0,0 +1,50 @@
+#include "AboutDialog.h"
+
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QPixmap>
+#include <QString>
+#include <QVBoxLayout>
+
+AboutDialog::AboutDialog(QWidget *parent)
+	: QDialog(parent)
+{
+	setWindowTitle(QStringLiteral("About RenderDoc"));
+
+	QLabel *letterhead = new QLabel;
+	letterhead->setPixmap(QStringLiteral(":/Resources/128.png"));
+
+	QLabel *version = new QLabel;
+	version->setText(QString("Version %1").arg("v0.28"));
```

Comment:
The version is stored in a couple of places, although I try and repeat it as few times as possible. Currently the 'canonical' version numbers are in `version.h` in the renderdoc project as `#defines`.

How about adding a query to the replay UI (in `renderdoc_replay.h` - with the misc functions at the bottom) that queries out major and minor version?

Ultimately we'll probably need to do the same as renderdocui, and repeat the version numbers somewhere in qrenderdoc for baking into the exe version information on windows, but for now it's nice not to repeat the version information in more places than we have to.